### DISCUSSION TITLE
Enabling user policies except on one column?

### DISCUSSION THREAD

**Discussion by ashiva** [supabase/supabase#2881] [Questions] [answered]
I have a Auth policy where users can only update their own records in a table if:

`(uid() = user_id)
`
However, I have a column for "views" on the the same table which I would like to update if anyone visits a page, not just the user. Is there a policy expression that would work for this or do I have to create a separate table for view count? 

Thanks

**Comment by inian** [comment]
Hi @ashiva, yup create a separate table for views seems cleaner here.

**Comment by steve-chavez** [comment, marked as answer]
@ashiva RLS always applies to the whole row. Check some strategies for a single column on this previous discussion: https://github.com/supabase/supabase/discussions/656#discussioncomment-335362
